frontend/app/src/components/history/events/forms/EvmEventForm.vue

```diff
@@ -498,6 +512,15 @@ const addressSuggestions = computed(() =>
         <VExpansionPanelContent
           class="[&>.v-expansion-panel-content\_\_wrap]:!p-0"
         >
+          <VTextField
```

Comment:
Is there any reason we went with VTextField instead of RuiTextField here?

Reply:
no, let me change

Reply:
updated